fast64.json

```diff
@@ -0,0 +1,62 @@
+{
+  "version": 1.0,
+  "autoLoad": true,
+  "microcode": "F3DEX2/LX2",
+  "rdpDefaults": {
+    "geometryMode": {
+      "zBuffer": true,
+      "shade": true,
+      "cullBack": true,
+      "lighting": true,
+      "shadeSmooth": true
+    },
+    "otherModeH": {
+      "textureConvert": "G_TC_FILT",
+      "textureFilter": "G_TF_BILERP",
+      "perspectiveCorrection": "G_TP_PERSP",
+      "pipelineMode": "G_PM_1PRIMITIVE"
+    },
+    "otherModeL": {},
+    "other": {}
+  },
+  "sm64": {
+    "draw_layers": {
+      "0": {
```

Comment:
There's no guarantee that layers are numbered from 0 to 7 when silhouette is on: https://github.com/HackerN64/HackerSM64/blob/master/include/sm64.h#L60 Use layer names instead for declaration if possible.

Reply:
Yep, that's exactly what the above comments are addressing 👍

Reply:
Worth noting too at some point the decal layers were moved in front of the standard layers, so this would've technically been incorrect even without decals. I think the new layout will still have an option for layer index which atm won't actually do anything (somebody correct me if I'm wrong).

Reply:
> There's no guarantee that layers are numbered from 0 to 7 when silhouette is on: https://github.com/HackerN64/HackerSM64/blob/master/include/sm64.h#L60 Use layer names instead for declaration if possible.

This is for the fast64 side, but yes the naming in the repo settings is bad (even if the internal props are by number) and that was adressed in the pr i mentioned along with giving the user full control of what draw layers they have access to in their fast64 materials
![image](https://github.com/user-attachments/assets/a6e332c9-7166-424b-b686-099b23c4e031)
![image](https://github.com/user-attachments/assets/74249523-f31e-4cc9-93d2-8673ac85f6a0)

Hackersm64 could actually add new draw layers with this!